mitosheet/src/mito/utils/actions.tsx

```diff
@@ -1094,6 +1094,32 @@ export const createActions = (
             searchTerms: ['steps', 'history'],
             tooltip: "View a list of all the edits you've made to your data."
         },
+        [ActionEnum.OpenSearch]: {
+            type: ActionEnum.OpenSearch,
+            shortTitle: 'Search',
+            longTitle: 'Search',
+            actionFunction: () => {
+                // We turn off editing mode, if it is on
+                setEditorState(undefined);
+                if (uiState.currOpenSearch.isOpen) {
+                    document.getElementById('mito-search-bar-input')?.focus();
```

Comment:
Almost. The problem is that if there are two mito sheets, this won't get the right one in every case. 

You can use the `mitoContainerRef` that is passed to this function, and then search inside that though - and this will work!